pages/app-layout/with-drawers.page.tsx

```diff
@@ -0,0 +1,151 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState } from 'react';
+import {
+  AppLayout,
+  ContentLayout,
+  Header,
+  HelpPanel,
+  NonCancelableCustomEvent,
+  SpaceBetween,
+  SplitPanel,
+  Toggle,
+} from '~components';
+import appLayoutLabels from './utils/labels';
+import { Breadcrumbs, Containers } from './utils/content-blocks';
+import ScreenshotArea from '../utils/screenshot-area';
+
+export default function WithDrawers() {
+  const [activeDrawerId, setActiveDrawerId] = useState<string | null>(null);
+  const [useDrawers, setUseDrawers] = useState(true);
+  const [useTools, setUseTools] = useState(true);
+  const [isToolsOpen, setIsToolsOpen] = useState(false);
+  const [drawersWidth, setDrawersWidth] = useState(400);
+
+  const drawers = !useDrawers
+    ? null
+    : {
+        drawers: {
+          activeDrawerId: activeDrawerId,
+          items: [
+            {
+              ariaLabels: {
+                closeButton: 'Security close button',
+                content: 'Security drawer content',
+                triggerButton: 'Security trigger button',
+                resizeHandle: 'Security resize handle',
+              },
+              content: <Security />,
+              id: 'security',
+              resizable: true,
+              size: drawersWidth,
+              onResize: (event: NonCancelableCustomEvent<{ size: number }>) => {
+                setDrawersWidth(event.detail.size);
+              },
```

Comment:
It probably should have been called during API review, but we need to move `onResize` on the top level, along with `onChange`.

This will allow to make `items` array fully static

Reply:
The idea here is that each panel / drawer can separately be resizable, and have separate sizes if the user wants. Is that still possible if `onResize` moves up a level?

Reply:
Yes, if event detail will also include drawer id. Then it will be closer to how resizable columns API is implemented in table